client/components/searchComponents/search.tsx

```diff
@@ -39,7 +39,7 @@ const Search: React.FC<SearchProps> = ({
         axios
             .get(`${backend_uri}/search`, {
                 params: { query },
-                headers: { Authorization: token },
+                headers: { token },
```

Comment:
This will break stuff, re-add Authorization

Reply:
Thought I did

Reply:
Fixed